Source/ThirdParty/gltfDefaults.js

```diff
@@ -69,10 +69,14 @@ define([
         }
```

Comment:
Since rotations are now quaternions, we also need to change
```javascript
if (!defined(node.rotation)) {
    // GLTF_SPEC: What is the default?  https://github.com/KhronosGroup/glTF/issues/197
    node.rotation = [1.0, 0.0, 0.0, 0.0];
}
```
to
```javascript
if (!defined(node.rotation)) {
    node.rotation = [0.0, 0.0, 0.0, 1.0];
}
```